prime-router/docs/design/proposals/0025-handling-missing-data.md

```diff
@@ -0,0 +1,60 @@
+# Proposal for handling missing timezones in messages
+
+## Background
+
+ReportStream is currently facing an ongoing issue regarding missing timezones in date times sent by some senders. 
+Certain STLTs require the conversion of all datetime values to their respective local timezones. In cases where 
+timezones are absent, ReportStream defaults the datetime to UTC and then converts it to the receiver's local timezone.
+This can be an issue since ReportStream is essentially making up data and could be reporting incorrect date/time(s).
+To tackle this issue, there are alternative approaches to handling missing timezones instead of resorting to UTC 
+as the default. We also propose implementing a logging mechanism to notify the STLT when such discrepancies occur.
+
+## Possible Implementations
+
+We should encourage Senders to include timezones for date times whenever they're missing from messages and issue 
```

Comment:
This sounds like validation logic that would directly contradict some HL7v2 profiles. Some profiles allow for date times without timezones, I think, and so validation would technically pass with NIST (which is presumably what the sender should only care about, unless we challenge that assumption).